packages/main/src/Table.ts

```diff
@@ -279,6 +280,17 @@ class Table extends UI5Element {
 	@property()
 	stickyTop = "0";
 
+	/**
+	 * Defines the height of the component.
+	 *
+	 * **Note:** When using sticky headers, make sure that the height of the table is defined.
+	 *
+	 * @default "auto"
+	 * @public
+	 */
+	@property()
+	height = "auto";
```

Comment:
I am not sure if i liked this property, i guess you know why.

Reply:
Removed it for now. Leads to issues if you set the table to Scroll mode (because of overflow-x: auto)